Core/wbDefinitionsSF1.pas

```diff
@@ -9115,32 +9115,45 @@ procedure DefineSF1b;
           wbActivityTracker
         ], []),
         wbRStruct('Component Data', [
-          wbArray(BUO4, 'Unknown',
-            wbStruct('Unknown', [
+          wbArray(BUO4, 'Blue Print Components',
+            wbStruct('Item', [
               wbFormIDCk('Base Item', [GBFM]),
               wbFormIDCk('Construction Object', [COBJ]),
               wbStruct('Position Offset', [
                 wbFloat('X'),
                 wbFloat('Y'),
                 wbFloat('Z')
               ]),
-              wbStruct('Unknown', [
-                wbFloat,
-                wbFloat,
-                wbFloat
+              wbStruct('Rotation', [
+                wbFloat('X'),
+                wbFloat('Y'),
+                wbFloat('Z')
```

Comment:
What units are the rotations specified in? if it isn't degrees, could use the same approach as is used in REFR/DATA

Reply:
radians